include/z64.h

```diff
@@ -737,8 +737,8 @@ typedef struct {
     /* 0x242 */ s16 heartsEnvR[2];
     /* 0x246 */ s16 heartsEnvG[2];
     /* 0x24A */ s16 heartsEnvB[2];
-    /* 0x24E */ s16 unk_24E;
-    /* 0x250 */ s16 unk_250;
+    /* 0x24E */ s16 health;
+    /* 0x250 */ s16 unk_timer;
```

Comment:
```suggestion
    /* 0x250 */ s16 unkTimer;
```
You can use Camel case instead of snake case when labelling variables